app/assets/javascripts/chart-data-mixin.js

```diff
@@ -0,0 +1,129 @@
+angular.module('containerDashboard').factory('ChartsDataMixin', ['$timeout', '$q', function chartDataMixinFactory ($timeout, $q) {
```

Comment:
please try to see what is relevant in this file and what's not. currently most of it is not needed